doc/userGuide.md

```diff
@@ -311,6 +311,47 @@ openjdk-tests/test-results/openjdk/report
 
 The JTREG report HTML summary file is then located at `openjdk-tests/test-results/openjdk/report/html/index.html`
 
+
+
+### Use live_monitor feature
+TKG has a script that monitors the test progress live. It can be your only friend in darkest nights. When you run tests locally, you may want to use that feature. It shows how many tests are passed currently etc. Now let's look how to use it.You need Python installed to your machine in order to use it. Note that this feature currently works with openjdk tests only. This means you need to use:	
```

Comment:
Please add a space between the two sentences: `Now let's look how to use it. You need Python installed to your machine in order to use it.`

Reply:
Done